src/main/kotlin/next/step/racing/domain/Cars.kt

```diff
@@ -0,0 +1,27 @@
+package next.step.racing.domain
+
+data class Cars(val cars: List<Car>) {
+    companion object {
+        fun init(count: Int): Cars {
+            return Cars(IntRange(1, count).map { Car.init() })
+        }
+
+        fun at(count: Int, pos: Position): Cars {
+            return Cars(IntRange(1, count).map { Car.at(pos) })
```

Comment:
아래처럼도 사용 가능할것 같네요. 반개구간(하나만 열린)을 원하시면 `(1 until count)`와 같이 사용하시면 됩니다.
```suggestion
            return Cars((1..count).map { Car.at(pos) })
```